README.md

```diff
@@ -1,4 +1,106 @@
+
+This is a little confusing, so let's build out our own example together. 
+
+### Code Along I: Relating Cats to Owners
+
+
+Let's revisit our `pets_database.db` and our `cats` table. Cd into the directory that contains your pet's database and type: `sqlite3 pets_database.db` to connect to our database. 
+
+#### Step 1: Creating the Owners Table
+
+First, we need to create our owners table. An owner should have an ID that is a primary key integer and a name that is text: 
+
+```sql
+sqlite> CREATE TABLE owners (id INTEGER PRIMARY KEY, name TEXT);
+```
+
+Now that we have our owners table, we can add a foreign key column to the pets table. 
+
+#### Step 2: Add Foreign Key to Pets Table
+
+Use the following statement to add this column: 
+
+```sql
+ALTER TABLE cats ADD COLUMN owner_id INTEGER;
+```
+
+Check your `cats` schema with `.schema` and you should see the following: 
+
+```sql
+CREATE TABLE cats (
+id INTEGER PRIMARY KEY,
+name TEXT,
+age INTEGER,
+breed TEXT,
+owner_id INTEGER);
+```
+
+Great, now we're ready to associate cats to their owners by creating an owner and assigning that owner's ID to certain cat's `owner_id` column. 
+
+#### Step 3: Associating Cats to Owners
+
+First, let's make a new owner: 
+
+```sql
+INSERT INTO owners (name) VALUES ("mugumogu");
+```
+
+Check that we did that correctly with the following statement: 
+
+```sql
+SELECT * FROM owners;
+```
+
+You should see the following: 
+
+
+```sql
+1 | mugumogu
+```
+
+Mugumogu is the owner of both Hana and Maru. Let's create our associations: 
+
+```sql
+UPDATE cats SET owner_id = 1 WHERE name = "Maru";
+UPDATE cats SET owner_id = 1 WHERE name = "Hana";
+```
+
+Let's check out our updated `pets` table: 
+
+```sql
+SELECT * FROM cats WHERE owner_id = 1;
+```
+
+This should return:
+
+```sql
+1 |Maru  | 3 |Scottish Fold| 1
+2 | Hana | 1 | Tabby       | 1
+```
+
+### Establishing Foreign Key: Determining Which Table Gets a "foreign key" Column
+
+Why did we decide to give our `pets` table the foreign key column and not the `owners` table? Similarly, in the example from the beginning of this exercise, why would we give a `posts` table a foreign key of `user_id` and not the other way around? 
+
+The answer lies in our understanding of relationships. In these instances, we have what is called a "has many" and "belongs to" relationship. An owner can have many cats, but (for the purposes of this example), a cat can only belong to one owner. Similarly, an user can have many posts that they've written but each post was written by and therefore belongs to only one user. 
+
+The thing that "has many" is considered to be the parent. The thing that "belongs to" we'll call the child. The child table gets the foreign key column, the value of which is the primary key of that data's/row's parent. 
```

Comment:
@SophieDeBenedetto what do you think about adding something about not wanting to grow a table horizontally? ie: 

If our owners table has to keep track of it's cat's ids, we'll need to continually add columns to accommodate this. If one owner had twelve cats, we'd need 12 columns called `cat_id_1`, `cat_id_2` etc. Owners with less than 12 cats would have empty columns. Since a cat has only one owner, they can keep track of that owner with a single column. Our tables then can grow vertically instead of horizontally.